otter/test/test_metrics.py

```diff
@@ -222,7 +221,8 @@ def test_added(self):
                 ServiceType.CLOUD_METRICS_INGEST, "POST", "ingest",
                 data=req_data, log=log).intent, noop)
         ]
-        eff = add_to_cloud_metrics(m['ttlInSeconds'], 'ord', metrics, 3, log)
+        eff = add_to_cloud_metrics(m['ttlInSeconds'], 'ord', metrics, 3,
```

Comment:
I agree with `3` seeming magical which is why I've added comment here. However, I am not so sure about making this kwarg due to this reason. It does help in increasing the readability but it also raises question about using positional args in general. When is it ok to have positional arg and when does it make the code unreadable. I think I'll keep this as it is unless you have a _very_ strong feeling against it. Thanks for the suggestion though :)